src/components/pages/Landing/RenderLandingPage.js

```diff
@@ -1,7 +1,17 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+import { Link, useHistory } from 'react-router-dom';
+
+
 
 function RenderLandingPage(props) {
+
+  const history = useHistory();
+
+  const loginHandler = () =>{
+    history.push('/login');
+  };
+
+
```

Comment:
Is this intended to be reusable or have anything added to it later? or would it be good to do inline on line 27 instead?